templates/zerver/register.html

```diff
@@ -81,7 +81,7 @@
         <div class="input-box">
             <div class="inline-block relative">
                 <input id="id_team_name" class="required" type="text"
-                    placeholder="Acme"
+                    placeholder="Äcmé"
```

Comment:
Changing my mind back to A without the umlaut (I think it looks like we're trying too hard with both :)).